src/components/Tabs.tsx

```diff
@@ -0,0 +1,39 @@
+import classNames from 'classnames';
+import React from 'react';
+import './Tabs.scss';
+
+interface Props {
+  selectedTab: Tab
+  tabs: Tab[],
+  onTabSelected: (tab: Tab) => void,
+}
+
+export const Tabs: React.FC<Props> = (props) => {
+  const { selectedTab, tabs, onTabSelected } = props;
+
+  return (
+    <div className="Tabs">
+      <ul className="Tabs__list">
+        {tabs.map(tab => (
+          <li key={tab.id} className="Tabs__items">
+            <button
+              type="submit"
```

Comment:
```suggestion
              type="button"
```
but better use link here